ui/pages/confirm-transaction-base/transaction-alerts/transaction-alerts.js

```diff
@@ -105,6 +106,13 @@ const TransactionAlerts = ({
           type="warning"
         />
       )}
+      {isNetworkBusy ? (
```

Comment:
This warning applies to the EIP-1559 V2 UI and will appear on the confirm transaction screen. It will NOT appear within the edit gas modal because we already have the status slider which indicates the status of the network.